e2e/page.ts

```diff
@@ -22,6 +24,12 @@ export default class PlaywrightDevPage {
         await this.page.goto(`${baseURL}/${defaultTeam}/messages/@${username}`);
     }
 
+    async createAndGoToGM(userIDs: string[]) {
+        const channel = await apiCreateGroupChannel(this.page.request, userIDs);
+        await this.page.goto(`${baseURL}/${defaultTeam}/channels/${channel.name}`);
+        return channel;
+    }
```

Comment:
Is there a reason we are not creating GMs in advance like the rest of resources?

Reply:
Well, it was originally because this was easier. Then I converted it (see the commit), but realized we still need the channel name and id in various places, so we still need an api call for each test to get that. Hopefully this `/group/search` solution won't break....